tests/v2/validation/nodescaling/scaling_node_driver_rke1_test.go

```diff
@@ -65,10 +65,10 @@ func (s *RKE1NodeScalingTestSuite) TestScalingRKE1NodePools() {
 		nodeRoles nodepools.NodeRoles
 		client    *rancher.Client
 	}{
-		{"Scaling control plane pool by 1", nodeRolesEtcd, s.client},
-		{"Scaling etcd node pool by 1", nodeRolesControlPlane, s.client},
-		{"Scaling worker node pool by 1", nodeRolesWorker, s.client},
-		{"Scaling worker node pool by 2", nodeRolesTwoWorkers, s.client},
+		{"Scaling control plane machine pool by 1", nodeRolesControlPlane, s.client},
```

Comment:
since this is rke1, I would suggest keeping it as `node pool` instead of machine.

Reply:
This is outdated, it was changed to remove pool completely in a separate PR Israel did.